### PR TITLE
make a sin method in java

### PR DESCRIPTION
**Pull request by Yash-Var**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:16]:
```suggestion
The **`Math.sin()`** method returns the sine of an angle argument, given in radians. The result is in the range [-1,1].
```

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:25]: Remove this header so the below is part of the Syntax section.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var, I did a first review and left a few comments.

**Comment by Yash-Var** [comment]
@SSwiniarski i have made changes ps review and merge it...

**Comment by Yash-Var** [comment]
@SSwiniarski 
ps review

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:25]:
```suggestion
```

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:28]:
```suggestion
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
A couple of more comments before it goes to a second review. Just need to remove a pair of line breaks.

**Comment by Yash-Var** [comment]
@SSwiniarski done

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:25]:
```suggestion
- Returns the `trigonometric` sine of the specified angle.
```

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/sin/sin.md:26]:
```suggestion
- Returns `NaN` if the specified angle is NaN or infinity.
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Yash-Var just two more things.

**Comment by Yash-Var** [comment]
@SSwiniarski done

**Review by SSwiniarski — COMMENTED** [review]
@Yash-Var This looks good for a second review.

**Review by SSwiniarski — APPROVED** [review]

**Comment by Yash-Var** [comment]
@jakemhiller  ps review and merge it ...

**Comment by Yash-Var** [comment]
@SSwiniarski ps merge it .......

**Comment by SSwiniarski** [comment]
@Yash-Var before it's merged, it needs to undergo another review by one of the maintainers.

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sin/sin.md:47]:
```suggestion
```shell
```

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sin/sin.md:16]:
```suggestion
The **`Math.sin()`** method returns the trigonometric sine of an angle argument, given in radians.
```

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sin/sin.md:26]:
```suggestion

The `angle` parameter is expressed in radians.

- The result is in the range [-1,1].
- If the `angle` is `NaN` or infinity, `NaN` is returned.
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Yash-Var Just had a few comments/suggestions for you. 😄

**Comment by Yash-Var** [comment]
@Dusch4593 done ps review.

File: content/java/concepts/math-methods/terms/sin/sin.md
Comment (Dusch4593) [inline content/java/concepts/math-methods/terms/sin/sin.md:43]:
```suggestion
public class Check {
  public static void main(String args[]) {
    // convert degrees to radians
    double pi = Math.PI;
    double degree = 60;
    double radian = degree * pi/180;
    System.out.println( "Sine of 60 degrees is " + Math.sin(radian));
  }
}
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@Yash-Var After we've implemented these last two suggestions, we should be good to merge. 😄

**Comment by Yash-Var** [comment]
@Dusch4593 done Ps review and merge it ..

**Review by Dusch4593 — APPROVED** [review]
@Yash-Var LGTM! 👍🏻

Could you catch up your `main` branch with the base branch so we can merge?

Screen Shot 2022-12-15 at 8 33 48 AM

**Comment by Yash-Var** [comment]
@Dusch4593 check.

**Comment by Yash-Var** [comment]
@Dusch4593 ps review and merge it...

**Comment by Dusch4593** [comment]
@Yash-Var Sorry, let's fetch/merge the base `main` branch one more time and I'll be on standby to merge. 😄

(I would do this on my end but I think you have a branch protection setting in place that only lets the original PR author do this)

**Comment by Yash-Var** [comment]
@Dusch4593  i have rebased the branch ps review..

**Comment by Yash-Var** [comment]
@Dusch4593 @SSwiniarski do you have more project in which i can contribute ...
ps help in finding the project in which i can contribute...

**Comment by KTom101** [comment]
@Yash-Var Feel free to take a look at the [issues tab](https://github.com/Codecademy/docs/issues) to see if there are any unclaimed issues you're interested in working on. 

Otherwise, if there is a specific topic that you noticed is missing from Codecademy then you can make a new issue for it. One of us will review it and let you know if it looks ok to add.

**Comment by yangc95** [comment]
@Yash-Var live link: https://www.codecademy.com/resources/docs/java/math-methods/sin
Screen Shot 2022-12-22 at 2 19 56 PM
Looks like you're all set with this entry, great work!

cc: @Dusch4593 @KTom101 @SSwiniarski

**Pull request merged**
